generate_pdf.py

```diff
@@ -97,8 +97,12 @@ def bgg_getter (command, params, config):
         status = a.status_code
         if(status != 200):
             error = ElementTree.fromstring(a.content)
-            print(error);
-            logging.info("Sleeping " + str(config.sleep_time) + " Seconds: " + (error.find('message').text if error != None else str(status)))
+            print(error)
```

Comment:
This line should probably be removed, actually. I added it earlier by accident while debugging the failure.